<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PipelineEditorWindow</class>
 <widget class="QMainWindow" name="PipelineEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pipeline Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="pipelineFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>350</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Modules</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listModules">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>331</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnShowAllModules">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>70</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Show All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHideAllModules">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>90</y>
       <width>70</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Hide All</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pipeline</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPipelineStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>70</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPipelineStop">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>30</y>
       <width>70</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPause">
     <property name="geometry">
      <rect>
       <x>162</x>
       <y>30</y>
       <width>70</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="videoFrame">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>581</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1258</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPipeline">
    <property name="title">
     <string>Pipeline</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuPipeline"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
